src/azure-cli/azure/cli/command_modules/hdinsight/_params.py

```diff
@@ -267,10 +267,10 @@ def load_arguments(self, _):
         # update
         with self.argument_context('hdinsight update') as c:
             c.argument('tags', tags_type)
-            c.argument('assign_identity', arg_group='Managed Service Identity', validator=validate_msi,
-                       completer=get_resource_name_completion_list_under_subscription(
-                           'Microsoft.ManagedIdentity/userAssignedIdentities'),
-                       help="The name or ID of user assigned identity.")
+            c.argument('assign_identity_type', arg_group='Managed Service Identity',
+                       help=("The type of identity used for the cluster. "
+                             "Allowed values: `None`, `SystemAssigned`, "
```

Comment:
why can the allowed value  be None?

Reply:
It can be skipped but no need to specify the None explicitly

Reply:
Pass in None to overwrite the existing identity of the cluster. This is also a needed scene.

Reply:
ok